regretBlocker.js

```diff
@@ -0,0 +1,9 @@
+// @flow
```

Comment:
Never used flow, so is this comment necessary?

Also, can we add 'we use flow' to the README somewhere so somewhere like me knows where to get started with it?